Dockerfile

```diff
@@ -1,21 +1,33 @@
-#FROM golang:1.15-buster AS builder
-#WORKDIR /go/src/sigs.k8s.io/ibm-powervs-block-csi-driver
-#COPY . .
-#RUN make
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
-FROM --platform=linux/ppc64le k8s.gcr.io/build-image/debian-base:v2.1.3 AS debian-base
-RUN clean-install ca-certificates e2fsprogs mount udev util-linux xfsprogs
-RUN clean-install bash multipath-tools sg3-utils
-# COPY --from=builder /go/src/sigs.k8s.io/ibm-powervs-block-csi-driver/bin/ibm-powervs-block-csi-driver /bin/ibm-powervs-block-csi-driver
-COPY ./bin/ibm-powervs-block-csi-driver /bin/ibm-powervs-block-csi-driver
+FROM golang:1.17 AS builder
+WORKDIR /go/src/sigs.k8s.io/ibm-powervs-block-csi-driver
+ADD . .
+ARG TARGETPLATFORM
+RUN ARCH=$(echo $TARGETPLATFORM | cut -f2 -d '/') make driver
 
-ENTRYPOINT ["/bin/ibm-powervs-block-csi-driver"]
+FROM k8s.gcr.io/build-image/debian-base:v2.1.3 AS debian-base
+RUN apt-get update -y \
```

Comment:
`apt-get update -y` is not really required, its already taken care here in the [clean-install script ](https://github.com/kubernetes/release/blob/db73544c45fba7f3c5d905a1b51ae1b81342ab39/images/build/debian-base/bullseye/clean-install#L28:L36)

Reply:
While testing it was not able to fetch some packages and suggested to `apt-get update`.
So it was added. I will try with out adding it once.

Reply:
removed